src/data/FilterCondition.py

```diff
@@ -0,0 +1,119 @@
+import shlex
+from MetaDataItem import MetaDataItem
+
+def is_string(token):
+    return (token[0] == '"' and token[-1] == '"') or (token[0] == "'" and token[-1] == "'")
```

Comment:
This would return true for `"asdf"asd"`

Reply:
Should be fixed now. The fix will also allow escaped quotes in strings, though the tokenizer is not currently set up to detect escaped characters.